apps/og-image/utils/zodParams.ts

```diff
@@ -52,15 +66,15 @@ export const fontParams = zodParams(
 
 export const challengeParam = zodParams(
   z.object({
-    title: truncatedWordSchema({ maxCharacters: 70 }),
-    description: truncatedWordSchema({ maxCharacters: 145 }),
-    username: z.string(),
+    title: truncateSchema({ maxCharacters: 70 }),
```

Comment:
i dont think you need to worry about og-image

you can move these zod utils to web

Reply:
ok

Reply:
remove all changes to og image pls

Reply:
aight